### PR TITLE
Create views using inline config

### PR DESCRIPTION
**Pull request by leeweisberger**
## Description
Create views using inline config, behind a feature flag.

Inline views are not created in a sub-folder. I think we want to drop this behavior.

Inline views require an `airplane` dependency for doing the inline configuraiton.

## Test plan
Manually tested. It would be pretty valuable to set up unit tests for this. We'd have to find a way to mock out the filesystem.

### CODE REVIEW

File: cmd/airplane/views/initcmd/init.go [hunk 1]
```diff
@@ -46,17 +51,24 @@ func New(c *cli.Config) *cobra.Command {
 			return Run(cmd.Root().Context(), cfg)
 		},
 	}
-	cmd.Flags().StringVar(&cfg.envSlug, "env", "", "The slug of the environment to query. Defaults to your team's default environment.")
```
Comment (leeweisberger) [inline cmd/airplane/views/initcmd/init.go:49]: We weren't using this

File: cmd/airplane/views/initcmd/init.go [hunk 2]
```diff
@@ -46,17 +51,24 @@ func New(c *cli.Config) *cobra.Command {
...
-func GetConfig(client *api.Client) config {
-	return config{client: client}
+func GetConfig(c *cli.Config) config {
+	return config{client: c.Client, root: c}
 }
 
 func Run(ctx context.Context, cfg config) error {
+	inlineSetByUser := cfg.cmd.Flags().Changed("inline")
+	if !inlineSetByUser {
+		if cfg.root.Flagger.Bool(ctx, logger.NewStdErrLogger(logger.StdErrLoggerOpts{}), flagsiface.DefaultInlineConfig) {
```
Comment (leeweisberger) [inline cmd/airplane/views/initcmd/init.go:68]: If the user doesn't set this value, the default is set via flag

File: cmd/airplane/views/initcmd/scaffolding/default.view.tsx [hunk 3]
```diff
@@ -45,13 +45,6 @@ const customersData = [
 ];
 
 const usersData = [
-  {
```
Comment (leeweisberger) [inline cmd/airplane/views/initcmd/scaffolding/default.view.tsx:48]: Unrelated... I just think there's too much data here and it always annoys me when i create a view :( I can revert if you like it

File: cmd/airplane/views/initcmd/scaffolding/default_inline.airplane.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    label: "Role",
+    accessor: "role",
+  },
+  {
+    label: "Email",
+    accessor: "email",
+  },
+];
+
+export default airplane.view(
```
Comment (leeweisberger) [inline cmd/airplane/views/initcmd/scaffolding/default_inline.airplane.tsx:107]: Exactly the same as the other... except for this

File: cmd/airplane/views/initcmd/init.go [hunk 5]
```diff
@@ -135,8 +155,10 @@ func createViewScaffolding(ctx context.Context, cfg *config) error {
 func generateEntrypointPath(cfg config, inViewDir bool) string {
 	if inViewDir {
 		return fmt.Sprintf("%s.view.tsx", cfg.slug)
-	} else {
+	} else if !cfg.inline {
 		return fmt.Sprintf("%s/%s.view.tsx", cfg.viewDir, cfg.slug)
+	} else {
+		return fmt.Sprintf("%s.airplane.tsx", cfg.slug)
```
Comment (leeweisberger) [inline cmd/airplane/views/initcmd/init.go:161]: I think we want to use this new format moving forward. Both work (.view.tsx and .airplane.tsx). According to @ameyakhare we made a decision to use .airplane.tsx and I tend to like that. We can also discuss in a thread if we want

**Review by rose-huang — APPROVED** [review]

**Pull request merged**